README.md

```diff
@@ -27,6 +21,18 @@ If you prefer to do it straight from CLI here is a one-liner for you:
 ```bash
 curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
 ```
+#### From source
+You can easily build OpenCompose tool from source. All you need is [Go installed](https://golang.org/dl/) on your system. Than simply run:
+```sh
+go get -u github.com/redhat-developer/opencompose
+```
+
+Although the simple `go get` installation works for just trying it out, if you want to submit bug reports you have to build it properly:
```

Comment:
I am ok with rephrasing if *properly* seem a bit too harsh, but the truth is that although `go get` works, it is not the supported way of building OpenCompose and you need to use `make install` (or `make build`) to have one that is supported. And of course all the binaries we build and publish on are release page are supported. (Supported means that we accept bug reports for it.)